### PR TITLE
Remove legacy repository

### PR DESCRIPTION
**Pull request by jdillon**
related:

https://github.com/sonatype/nexus-pro/pull/533
https://github.com/sonatype/nexus-bundles/pull/147

### CODE REVIEW

File: components/nexus-core/src/main/java/org/sonatype/nexus/internal/httpclient/UserAgentGenerator.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import org.sonatype.sisu.goodies.common.ComponentSupport;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Generates the {@code User-Agent} header value.
+ *
+ * @since 3.0
+ */
+@Named
```
Comment (cstamas) [inline components/nexus-core/src/main/java/org/sonatype/nexus/internal/httpclient/UserAgentGenerator.java]: Why not Singleton?

File: components/nexus-core/src/main/java/org/sonatype/nexus/internal/ApplicationDirectoriesImpl.java [hunk 2]
```diff
@@ -10,7 +10,7 @@
  * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
  * Eclipse Foundation. All other trademarks are the property of their respective owners.
  */
-package org.sonatype.nexus;
+package org.sonatype.nexus.internal;
```
Comment (cstamas) [inline components/nexus-core/src/main/java/org/sonatype/nexus/internal/ApplicationDirectoriesImpl.java]: See https://github.com/sonatype/nexus-oss/pull/1264

File: components/nexus-core/src/main/java/org/sonatype/nexus/internal/httpclient/UserAgentGenerator.java (hunk 1 above)
Comment (jdillon) [inline components/nexus-core/src/main/java/org/sonatype/nexus/internal/httpclient/UserAgentGenerator.java]: left over when this was a javax.inject.Provider, should be @Singleton

File: components/nexus-core/src/main/java/org/sonatype/nexus/internal/ApplicationDirectoriesImpl.java (hunk 2 above)
Comment (jdillon) [inline components/nexus-core/src/main/java/org/sonatype/nexus/internal/ApplicationDirectoriesImpl.java]: yes I saw and one of us will have fun time resolving merge conflicts

**Comment by djsauble** [comment]
Do all the …/coreui/view/routing/* views get nuked as well?

**Comment by jdillon** [comment]
@djsauble yes probably a few other straggling bits too I've not killed as well

**Comment by jdillon** [comment]
manually merged

**Pull request closed without merging**